requests_toolbelt/adapters/source.py

```diff
@@ -28,7 +28,10 @@ class SourceAddressAdapter(HTTPAdapter):
         s.mount('http://', SourceAddressAdapter('10.10.10.10'))
     """
     def __init__(self, source_address, **kwargs):
-        self.source_address = source_address
+        if isinstance(source_address, basestring):
+            self.source_address = (source_address, 0)
+        else:
```

Comment:
I'm not particular about it but I want to prevent future problems.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.